paddle/fluid/framework/details/inplace_op_pass.cc

```diff
@@ -148,12 +150,14 @@ std::unique_ptr<ir::Graph> InplacePass::ApplyImpl(
   view_.Build(graph.get());
   InitSSAGraphNodes();
 
+  auto cnt = 0;
   for (auto* op : view_.AllOps()) {
+    VLOG(4) << "Handle op " << cnt++ << ": " << op->Name();
     if (FLAGS_enable_inplace_whitelist && !whitelist_.count(op->Name()))
       continue;
     TryInplaceOpInputOutput(op, graph.get());
   }
-  graph->ResolveHazard(var_nodes_);
+  // graph->ResolveHazard(var_nodes_);
```

Comment:
why?

Reply:
> why?

in-place logic should not affect the executing logic of the graph, I think it's meaningless to add ResolveHazard here.